lib/logstash/codecs/csv.rb

```diff
@@ -108,10 +113,41 @@ def register
     @logger.debug? && @logger.debug("CSV parsing options", :col_sep => @separator, :quote_char => @quote_char)
   end
 
-  def decode(data)
-    data = @converter.convert(data)
+  def decode(data, &block)
+    puts("*** data=#{data.inspect}")
+    @buffer.extract(data).each do |line|
+      puts("*** line=#{line.inspect}")
```

Comment:
stray `puts` statements
```suggestion
    @buffer.extract(data).each do |line|
```